pkg/resources/password.go

```diff
@@ -0,0 +1,47 @@
+package resources
+
+import (
+	"math/rand"
+	"strings"
+)
+
+var (
+	lowerCharSet   = "abcdedfghijklmnopqrst"
```

Comment:
Not sure how much this matters but the character `d` is listed twice in the string `lowerCharSet`

Reply:
updated 👍